### PR TITLE
Copyedit on the Philadelphia story

### PR DESCRIPTION
**Pull request by randomecho**
- Officer as position, not Office
- "As if this all wasn't awesome enough" was a little hard to parse. Hoping this version catches the awesome in the same voice.
- Missing "of" in City of Philadelphia

### CODE REVIEW

File: _posts/2013-09-26-philadelphia-gets-going-and-gets-open.md [hunk 1]
```diff
@@ -7,16 +7,16 @@ image: /images/philadelphia.png
...
 
-As if this all wasn't awesome enough, Headd and his team have posted their [Open Government Plan](https://github.com/CityOfPhiladelphia/open-gov-phl) and are using GitHub Issues to get feedback and data requests _and_ are using GitHub to experiment with better procurement. You can read his overview of the process [here](http://civic.io/2013/03/27/experiments-in-github-based-procurement/).
+The awesome doesn't stop there as Headd and his team have posted their [Open Government Plan](https://github.com/CityOfPhiladelphia/open-gov-phl) and are using GitHub Issues to get feedback and data requests _and_ are using GitHub to experiment with better procurement. You can read his overview of the process [here](http://civic.io/2013/03/27/experiments-in-github-based-procurement/).
```
Comment (afeld) [inline _posts/2013-09-26-philadelphia-gets-going-and-gets-open.md]: I think I liked this line better before.  "The awesome" doesn't quite fit the voice of the rest of the post.

**Comment by jlord** [comment]
Thanks @randomecho! I'm :thumbsup: with the changes but agree with @afeld re: the awesome line. Maybe change the orig to "As if all _of_ this _work_ wasn't awesome enough," (emphasis = additions)

**Comment by randomecho** [comment]
Good call @jlord and @afeld and have leaned in with the suggested update to the updated awesome line.

**Comment by jlord** [comment]
Thanks so much @randomecho!!
